testcases/OpTestInterrupt.py

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/env python3
+# IBM_PROLOG_BEGIN_TAG
+# This is an automatically generated prolog.
+#
+# $Source: op-test-framework/testcases/OpTestXive.py $
+#
+# OpenPOWER Automated Test Project
+#
+# Contributors Listed Below - COPYRIGHT 2017
+# [+] International Business Machines Corp.
+#
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing
+# permissions and limitations under the License.
+#
+# IBM_PROLOG_END_TAG
+
+'''
+OpTestInterrupt
+---------------
+
+This module contain testcases related to XIVE.
+
+1.Enable kernel logging, basically generate traffic
+2.Enable console traffic by printing the 
+processes/tasks to the console
+3.check Xive is configured properly or not
+4.check system reboot with xive=on
+5.check system reboot with xive=off
+6.validate interrupts handled by CPU's
+'''
+import os
+import re
+import sys
+import time
+import pexpect
+import unittest
+import subprocess
+
+import OpTestConfiguration
+from common import OpTestHMC, OpTestFSP
+from common.OpTestSOL import OpSOLMonitorThread
+from common.OpTestError import OpTestError
+from common.OpTestSystem import OpSystemState
+from common.OpTestSSH import ConsoleState as SSHConnectionState
+from common.OpTestConstants import OpTestConstants as BMC_CONST
+from common.Exceptions import KernelOOPS, KernelKdump
+
+import logging
+import OpTestLogger
+log = OpTestLogger.optest_logger_glob.get_logger(__name__)
+
+
+class OpTestInterrupt(unittest.TestCase):
+    def setUp(self):
+        conf = OpTestConfiguration.conf
+        self.cv_SYSTEM = conf.system()
+        self.cv_HOST = conf.host()
+        self.platform = conf.platform()
+        self.bmc_type = conf.args.bmc_type
+        self.util = self.cv_SYSTEM.util
+        self.c = self.cv_SYSTEM.console
+        self.con = self.cv_SYSTEM.cv_HOST.get_ssh_connection()
+    
+    def xive_test(self):
+        '''
```

Comment:
All you are doing is enabling and writing to console. Consider having better function  name.